fuzz/asn1.c

```diff
@@ -335,7 +335,7 @@ int FuzzerTestOneInput(const uint8_t *buf, size_t len)
     DO_TEST_NO_PRINT(DH, d2i_DHparams, i2d_DHparams);
     DO_TEST_NO_PRINT(DH, d2i_DHxparams, i2d_DHxparams);
 #endif
-#ifndef OPENSSL_NO_DSA
+#if !defined(OPENSSL_NO_DSA) || !defined(OPENSSL_NO_DEPRECATED_3_0)
     DO_TEST_NO_PRINT(DSA_SIG, d2i_DSA_SIG, i2d_DSA_SIG);
```

Comment:
Should be outside of the deprecated ifdef.

Reply:
Fixed